mode_manager_cli.py

```
@@ -0,0 +1,56 @@
+            if not mm.is_build_mode():
+                print("Not in build mode. Use /mode build first.")
+                continue
+            _, file_path = cmd.split(maxsplit=1)
+            try:
+                orig = open(file_path, encoding="utf-8").read()
+            except Exception:
+                orig = ""
+            print("Enter new content (end with a single line containing only 'END'):")
+            lines = []
+            while True:
+                l = input()
```

Ruff (0.12.2)

Ambiguous variable name: `l`

(E741)